sparsity/sparse_frame.py

```diff
@@ -108,9 +108,10 @@ def todense(self, pandas=True):
         else:
             dense = np.empty(shape=(0, len(self.columns)))
 
-        if self.shape[0] == 1 or self.shape[1] == 1:
-            dense = dense.reshape(-1)
-        if pandas == True:
+        if pandas:
+            if self.shape[0] == 1 or self.shape[1] == 1:
+                dense = dense.reshape(-1)
+
```

Comment:
Because then we have the correct behaviour when using multiply with SparseFrames, which uses toarray() function. If we pass 1 dimensional array we should then specify how to multiply (column or row wise). I saw that you suggested that in the next comment. With current setting we can do: `sf.multiply(sf['col1'])` to multiply colwise. If `sf['col1'].toarray()` is 1-dim this doesn't work.